### ISSUE TITLE
Bug in Summernote Editor or contact form?

### ISSUE THREAD

**Issue by TechArea** [robiso/wondercms#156]
Hi

I've noticed an error in Summernote editor in combination with the contact form.

The error is on the page end.

`Notice: Undefined variable: script in /home/techarea/public_html/plugins/summernote/summernote.php on line 52`

PS: I'm using the function to display the contact form a specific page.

**Comment by StephanStanisic** [comment]
Hi @TechArea,

This is a known bug and has been fixed in the experimental 3.0.0 release of WonderCMS.

To fix this in wondercms to could either disable errors in a .htaccess file:
```
php_flag display_errors off
```

Or to fix the actual issue, move the `$args[0].=$script` in `plugins/summernote/summernote.php` to one line higher:
```
function loadSummerNoteJS($args) {
    if (wCMS::$loggedIn) {
        $script = <<<'EOT'
        <script src="https://cdn.jsdelivr.net/npm/summernote@0.8.8/dist/summernote.min.js" integrity="sha384-rv18XCwOLyh2rO/6Vz9eKWaSP+ljKfFed/npYlSg476id+996jfNDF+5iC3W5KKJ" crossorigin="anonymous"></script>
        <script src="plugins/summernote/js/admin.js"></script>
EOT;
        $args[0].=$script;
    }
    return $args;
}
 
function loadSummerNoteCSS($args) {
    if (wCMS::$loggedIn) {
        $script = <<<'EOT'
        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/summernote@0.8.11/dist/summernote.css" integrity="sha384-hHEPAd3Dkb316VuNPtIZ6LUrzxvD4PQOTW478Ww6c/aUJKXNDV9pEx5/jZgISR1G" crossorigin="anonymous">
        <link rel="stylesheet" href="plugins/summernote/css/admin.css" type="text/css" media="screen">
EOT;
        $args[0].=$script;
    }
    return $args;
}
```
Let me know if this fixes your issue!

**Comment by robiso** [comment]
@TechArea, closing issue, please feel free to reopen issue if necessary.

@StephanStanisic, thanks for helping out :)
